modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -235,6 +234,14 @@ export function StyleSheetLoader(documentOrShadowRoot: DomDocument | ShadowRoot,
     });
   };
 
+   /**
+   * Loads the specified css style sheet files and call the success callback once it's finished loading.
```

Comment:
Maybe replace `sheet files` with just `sheets`? What you have is fine though, just sounds a little bit weird.

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5821/commits/8116c5ddb68cd5b7f8f40f874b619996d09fd3c7

Reply:
replaced by https://github.com/tinymce/tinymce/pull/5821/commits/2a18505d219e9ad0fcb78a1865a2761f1d192592